src/main/java/org/springframework/retry/annotation/RecoverAnnotationRecoveryHandler.java

```diff
@@ -139,6 +158,8 @@ private boolean compareParameters(Object[] args, int argCount, Class<?>[] parame
 	private void init(Object target, Method method) {
 		final Map<Class<? extends Throwable>, Method> types = new HashMap<Class<? extends Throwable>, Method>();
 		final Method failingMethod = method;
+		Retryable retryable = method.getAnnotation(Retryable.class);
+		this.recoverMethodName = retryable != null ? retryable.recoverName() : "";
```

Comment:
I think `null` by default is OK, when we start to use a `StringUtils.isEmpty()` mentioned above

Reply:
Fixed.